usi/src/main/proto/usi.proto

```diff
@@ -54,6 +54,15 @@ enum POEStatus {
   DENY = 3;
 }
 
+message SwitchInput {
+  // Device Port
+  int32 device_port = 1;
+  oneof input {
+    SwitchInfo switch_info = 2;
```

Comment:
Something about this doesn't seem right. These are the arguments to each individual RPC request, right?  Why is switch_info passed along with each one?  Why does the caller explicitly know about the faux interface?  I think I'm missing some context on where this is used  on the calling side... where in the code is the RPC call constructed? (I see where the info is used)

Reply:
rpc call is constructed in DAQ in the other PR.

Reply:
I wanted to make these grpc calls which was why these switch information are passed every time. Theoretically, a single usi can be used to manipulate multiple switches for multiple DAQ instances.